challenges/objects-arrays.js

```diff
@@ -55,11 +80,19 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for(let i = 0;i < graduates.length;i++){
+  contactInfo.push(graduates[i].first_name + " " + graduates[i].email);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for(let i = 0;i < graduates.length;i++){
+ if(graduates[i].university.includes ("Uni")){
+   uni.push(graduates[i].university);
+ }
+}
 console.log(uni);
```

Comment:
Great! What other ways can this be solved? Try solving it using regular expressions :)